openkfw/TruBudget
# API: Refactor workflowitem.validateDocument

# Description

I've refactored the document validation code towards the new code layout. There's no automated test for this yet, so please make sure it works on your machine too.

Closes #143.

# How to test this

- Go to an open workflowitem and upload a document
- Use the (I) button to the left to show the document hash; use the "validate" button and select the file you've just uploaded - you should see "OK"
- Try the same with a different document - you should see "Not OK"
